packages/worker/src/core.ts

```diff
@@ -58,11 +73,46 @@ export class Core {
           : `http://${compiledServerOptions.address}`,
       },
       telemetryOptions,
+      logger: options.logger ?? new DefaultLogger('INFO'),
     };
     const native = await promisify(newCore)(compiledOptions);
+
     return new this(native, compiledOptions);
   }
 
+  private initLogPolling(options: CompiledCoreOptions, native: native.Core) {
+    this.shouldPollForLogs.next(true);
+
+    if (options.logger != null) {
```

Comment:
Is this even possible?
Don't we always want there to be a logger?

Reply:
Ah yeah I guess this isn't possible now. I do want it to be disable-able but I suppose that works just fine by setting forwarding level to off, so I'll check there instead.